src/video/windows/SDL_windowskeyboard.c

```diff
@@ -108,9 +108,9 @@ void WIN_InitKeyboard(_THIS)
     SDL_SetScancodeName(SDL_SCANCODE_RGUI, "Right Windows");
 
     /* Are system caps/num/scroll lock active? Set our state to match. */
-    SDL_ToggleModState(SDL_KMOD_CAPS, (GetKeyState(VK_CAPITAL) & 0x0001) != 0);
-    SDL_ToggleModState(SDL_KMOD_NUM, (GetKeyState(VK_NUMLOCK) & 0x0001) != 0);
-    SDL_ToggleModState(SDL_KMOD_SCROLL, (GetKeyState(VK_SCROLL) & 0x0001) != 0);
+    SDL_ToggleModState(SDL_KMOD_CAPS, (GetKeyState(VK_CAPITAL) & 0x0001));
```

Comment:
These expect a book, but I suppose these are okay.

Reply:
yes, I think this is ok, because tool is check to be not 0

```
113 void SDL_ToggleModState(const SDL_Keymod modstate, const SDL_bool toggle)
1114 {
1115     SDL_Keyboard *keyboard = &SDL_keyboard;
1116     if (toggle) {
```

Reply:
I was trying to scope the change to if statements. For casting to SDL_bool, the correct change is (x & y) ? SDL_TRUE : SDL_FALSE

Reply:
ok updated